server/planning/io/ingest_rule_handler.py

```diff
@@ -164,13 +164,6 @@ def add_planning_agendas(self, ingest_item: Dict[str, Any], attributes: Dict[str
 
     def process_autopost(self, ingest_item: Dict[str, Any]):
         """Automatically post this item"""
-
```

Comment:
This might lead to double publishes, once here and once in the update event or planning. That is why this was there in the first place.

Reply:
yep I've realized later it might be problematic, I think it would be more reliable to use a different field than pubstatus internally because we need a way to ingest pubstatus too.

Reply:
but looking at the code it would be probably easier to introduce some `ingest_pubstatus` and handle it from there, there is too much code using that `pubstatus` now

Reply:
Yeah let's do that. Quick fix that's solid, reducing risk of bugs